src/snowflake/snowpark/session.py

```diff
@@ -4114,6 +4114,7 @@ def call(
         *args: Any,
         statement_params: Optional[Dict[str, Any]] = None,
         log_on_exception: bool = False,
+        return_dataframe: Optional[bool] = None,
```

Comment:
for boolean, I think conventionally we just set default to False
```suggestion
        return_dataframe: bool = False,
```

Reply:
I think this is a different situation that we need to follow the behavior of internal `is_return_table`